### PR TITLE
Bug 1375034 Added Configuring a Regsitry Location

### PR DESCRIPTION
**Pull request by bmcelvee**
https://bugzilla.redhat.com/show_bug.cgi?id=1375034

### CODE REVIEW

File: install_config/install/disconnected_install.adoc [hunk 1]
```diff
@@ -489,6 +489,46 @@ registry.access.redhat.com/jboss-webserver-3/webserver30-tomcat7-openshift:lates
...
+----
+oreg_url=example.com/openshift3/ose-${component}:${version}
+openshift_examples_modify_imagestreams=true
+----
+
+If your registry is secure, you will also need to configure:
+----
+openshift_docker_additional_registries=$REGISTRY:5000
+openshift_docker_insecure_registries=$REGISTRY:5000
+----
```
Comment (jianlinliu) [inline install_config/install/disconnected_install.adoc]: I do not think all the registry is using 5000 port, how about correct the above lines like this:
Depends on your registry configuration, you maybe also need to configure:
openshift_docker_additional_registries=example.com
openshift_docker_insecure_registries=example.com

File: install_config/install/disconnected_install.adoc [hunk 2]
```diff
@@ -489,6 +489,46 @@ registry.access.redhat.com/jboss-webserver-3/webserver30-tomcat7-openshift:lates
...
+
+|`*openshift_docker_additional_registries*`
+|Set to the URL created with the docker images. Verify that the name and label of
+the repository match the original name.
+
+|`*openshift_docker_insecure_registries*`
+|Set to the URL created with the docker images. Verify that the name and label of
+the repository match the original name.
+|===
+
```
Comment (jianlinliu) [inline install_config/install/disconnected_install.adoc]: I do not think the explanation for openshift_docker_additional_registries and openshift_docker_insecure_registries are correct.

**Comment by bmcelvee** [comment]
@ahardin-rh peer review please 😸

File: install_config/install/disconnected_install.adoc [hunk 3]
```diff
@@ -489,6 +489,55 @@ registry.access.redhat.com/jboss-webserver-3/webserver30-tomcat7-openshift:lates
...
+|Variable |Purpose
+|`oreg_url`
+|Set to the alternate image location. Necessary if you are not using the default
+registry at `registry.access.redhat.com`.
+
+|`openshift_examples_modify_imagestreams`
+|Set to `true` if pointing to a registry other than the default. Modifies the
+image stream location to the value of `*oreg_url*`.
```
Comment (ahardin-rh) [inline install_config/install/disconnected_install.adoc]: `oreg_url`

File: install_config/install/disconnected_install.adoc [hunk 4]
```diff
@@ -489,6 +489,55 @@ registry.access.redhat.com/jboss-webserver-3/webserver30-tomcat7-openshift:lates
...
+
+|`openshift_docker_additional_registries`
+|Set `openshift_docker_additional_registries` to add it's value in the
```
Comment (ahardin-rh) [inline install_config/install/disconnected_install.adoc]: its value

File: install_config/install/disconnected_install.adoc [hunk 5]
```diff
@@ -489,6 +489,55 @@ registry.access.redhat.com/jboss-webserver-3/webserver30-tomcat7-openshift:lates
...
+`add_registry` line in `/etc/sysconfig/docker`. With `add_registry`, you can add
```
Comment (ahardin-rh) [inline install_config/install/disconnected_install.adoc]: *_/etc/sysconfig/docker_*

File: install_config/install/disconnected_install.adoc [hunk 6]
```diff
@@ -489,6 +489,55 @@ registry.access.redhat.com/jboss-webserver-3/webserver30-tomcat7-openshift:lates
...
+your own registry to be used for Docker search and Docker pull. Use the
+`add_registry` option to list a set of registries, each prepended with
+`--add-registry` flag. The first registry added will be the first registry
+searched. For example, `add_registry=--add-registry registry.access.redhat.com
+--add-registry example.com`.
+
+|`openshift_docker_insecure_registries`
+|Set `openshift_docker_insecure_registries` to add it's value in the
```
Comment (ahardin-rh) [inline install_config/install/disconnected_install.adoc]: its value

File: install_config/install/disconnected_install.adoc [hunk 7]
```diff
@@ -489,6 +489,55 @@ registry.access.redhat.com/jboss-webserver-3/webserver30-tomcat7-openshift:lates
...
+`insecure_registry` line in /etc/sysconfig/docker. If you have a registry
```
Comment (ahardin-rh) [inline install_config/install/disconnected_install.adoc]: *_/etc/sysconfig/docker_*

File: install_config/install/disconnected_install.adoc [hunk 8]
```diff
@@ -489,6 +489,55 @@ registry.access.redhat.com/jboss-webserver-3/webserver30-tomcat7-openshift:lates
...
+secured with https but do not have proper certs distributed, you can tell Docker
```
Comment (ahardin-rh) [inline install_config/install/disconnected_install.adoc]: HTTPS
s/certs/certificates

**Comment by ahardin-rh** [comment]
Just a few comments. Otherwise, LGTM!

**Pull request merged**